### PR TITLE
Can add new assignments using frontend

### PR DESCRIPTION
**Pull request by gosusleeper**

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1 +1,8 @@
 export const SERVER_URL = 'http://localhost:8081';
+
+
+export const SEMESTER_LIST = [
+    {id:0, year:2020, name:'Fall'},
+    {id:1, year:2021, name:'Spring'},
+    {id:2, year:2021, name:'Fall'}
+  ] ;
```
Comment (abellizzistudent) [inline src/constants.js]: "No newline at EOF" likely is not causing issues, but consider revising or reviewing in the future if problems arise.

**Comment by abellizzistudent** [comment]
Your code looks good to me. Review my comment on the EOL warning, OK to merge without resolving it for now, I don't think it is causing issues, just know it's there.

File: src/constants.js (hunk 1 above)
Comment (gosusleeper) [reply src/constants.js]: Thanks for the catch.

**Review by gosusleeper — COMMENTED** [review]
LGTM

**Pull request merged**